jedi/api/completion.py

```diff
@@ -84,7 +100,10 @@ def __init__(self, inference_state, module_context, code_lines, position,
         self._position = position[0], position[1] - len(self._like_name)
         self._call_signatures_callback = call_signatures_callback
 
-    def completions(self):
+    def completions(self, **kwargs):
+        return self._completions(**kwargs)
+
+    def _completions(self, method=start_match):
```

Comment:
Just use `fuzzy=False` here for now instead of `method`. This is IMO a good API decision. I'm still not sure if I want to make it the default.